### ISSUE TITLE
MatMenu open position not working with 2.4.3 - worked in 2.3

### ISSUE THREAD

**Issue by whentotrade** [SamProf/MatBlazor#552]
I was using the standard usage as in the docs top open at MatMenu from a IconButton like this:

```
<MatIconButton Icon="more_vert" OnClick="@ShowMenu" RefBack="@buttonForwardRef"></MatIconButton>
<MatMenu @ref="Menu" TargetForwardRef="@buttonForwardRef">
        <MatList>
                        <MatListItem OnClick="@(() => { ... })">Chart</MatListItem>
                        <MatListItem OnClick="@(() => { ... })">Scan</MatListItem>
          </MatList>
</MatMenu>

```
Code part:
```
ForwardRef buttonForwardRef = new ForwardRef();
BaseMatMenu Menu;

public void ShowMenu(MouseEventArgs e)
 {
        this.Menu.OpenAsync();
}
```

On click of the icon the menu opened at the position of the icon.

Now, the same code with 2.4.3, after the click on the icon the menu opens not at the position of the icon. The menu opens at the upper left corner of the HTML browser window.

Any idea what changed from 2.3 to 2.4.3 in that case? When rolling back to 2.3 everything looks fine again?

**Comment by whentotrade** [comment]
I will close this topic as it seems to be resolved with 2.6 release.

**Comment by mynkow** [comment]
I have the same issue when I place matmenu inside a table cell. Any clue?
